src/applications/disability-benefits/686/helpers.js

```diff
@@ -1,6 +1,8 @@
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
```

Comment:
You should change the extension for this file (helpers.js) to be jsx.

Reply:
k